README.md

```diff
@@ -28,13 +27,297 @@ pip install -U pyinstaller-hooks-contrib
 
 Either `a-package` works fine without a hook, or no-one has contributed hooks.
 If you'd like to add a hook, or view information about hooks,
-please see [the wiki](https://github.com/pyinstaller/pyinstaller-hooks-contrib/wiki).
+please see below.
 
 
 ## I want to help!
 
-Please start by providing pull requests and helping solve issues.
+If you've got a hook you want to share then great!
+The rest of this page will walk you through the process of contributing a hook.
+If you've been here before then you may want to skip to the [summary checklist](#summary)
+
+**Please provide one hook per pull request! If you have more than one then submit them in separate pull requests.**
+
+
+### Setup
+
+[Fork this repo](https://github.com/pyinstaller/pyinstaller-hooks-contrib/fork) if you haven't already done so.
+Clone and `cd` inside your fork by running the following (replacing `bob-the-barnacle` with your github username):
+
+```
+git clone git@github.com:bob-the-barnacle/pyinstaller-hoooks-contrib.git
```

Comment:
Do you think it would be more sensible to show an https url? I think someone who wants to use ssh is more likely to know to switch the url format than the other way around.

Reply:
It might be; if nothing else, GitHub also offers https url by default. Plus, if someone had set keys for ssh access, they will probably know to switch from https url to ssh one.